src/app/components/app.tsx

```diff
@@ -59,7 +59,7 @@ export const App = (props: IAppProps) => {
     initialInputState: defaultAuthoredState,
     initialOutputState: {
       time: 0,
-      soilChange: "--",
+      lightChange: "--",
```

Comment:
I know that you didn't author and it's coming from DESE sim, but I'd just point out that's probably not the best choice for a view-agnostic state. I'd rather use `null`, `undefined`, or `0`. This string is purely related to the view that might change at any point. And comparing state to "--" in random places is also pretty awkward. Enum could help too.

I don't think it's worth changing it at this point, just a random comment. ;)

Reply:
That's a good point! Thanks Piotr.

Also, sorry I didn't add more info to help contextualize the PR! Yes, you're right, the story was just to change over the look of the controls and the names / options, but it isn't affecting the model in any way (other than showing up in the table correctly). This is the story in PT: https://www.pivotaltracker.com/story/show/183629709